File: ADSO/Lenguajes/Dart/funciones_practica.dart
import 'dart:io';

void saludar(nombre, sexo){
  sexo = sexo.toLowerCase();
  var adjetivo;

  
  if (sexo == "mujer" ) {
    adjetivo = "Guapa";

  }else if (sexo == "hombre" ){
    adjetivo = "Guapisimo";

  }else {
    adjetivo = "tú";
  }

  print("hola $nombre ¿como andas $adjetivo?");
}

void main(){
  for (var i = 0; i < 3; i++) {
    print("Sexo:");
  String? a = stdin.readLineSync();
  print("Nombre:");
  String? name = stdin.readLineSync();
  
  saludar(name, a);
  
  print("-------------------------------------------------");
  }
}

File: ADSO/Lenguajes/Ruby/funciones_practica.rb
def saludar(nombre, sexo)
  sexo = sexo.downcase

  if sexo == "mujer"
    adjetivo = "mina"
  elsif sexo == "hombre"
    adjetivo = "maquina"
  else
    adjetivo = "guapo"
  end

  puts "Hola #{nombre}, ¿cómo andas #{adjetivo}?"
end

# Input
for m in 1..3

  puts "Sexo:"
  sexo = gets.chomp
  puts "Nombre:"
  nombre = gets.chomp

  saludar(nombre, sexo)
  puts ("-------------------------------------------------")
end
